CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to OpenReview
+
+Welcome, and thank you for your interest in contributing to OpenReview!
+
+The goal of this document is to provide a high-level overview of how you can get involved by reporting issues in OpenReview and its documentation.
+
+## Asking Questions
+
+The issue tracker is intended for bugs found in the system. If your question is related to how OpenReview works, rather than opening an issue, please first check our [FAQ](https://openreview.net/faq). If you cannot find the answer there, please contact the organizers of your venue or contact us at info@openreview.net.
+
+## Providing Feedback
+
+**Your comments and feedback are welcome**, and the development team is available via a handful of different channels.
+
+See the [Feedback Channels](https://github.com/microsoft/vscode/wiki/Feedback-Channels) wiki page for details on how to share your thoughts.
```

Comment:
this looks like a link to feedback to vscode

Reply:
Thanks! Fixed